docs/Data_Portal/Users_Guide/analysis_center.md

```diff
@@ -47,6 +47,6 @@ If there is not sufficient data in the active cohort to use a particular tool, t
 
 As each tool is selected, it is loaded in the 'Analysis Center' within a panel.
 
-[![Analysis Center Tools Panel](images/AnalysisCenterToolPanel.png)](images/AnalysisCenterToolPanel.png "Click to see the full image.")
+[![Analysis Center Tools Panel](images/CohortBuilderQuickStart.png)](images/CohortBuilderQuickStart.png "Click to see the full image.")
```

Comment:
I was looking at this image and I think we might have put the wrong image there in the first place (it should probably be one of the analysis tools rather than the cohort builder).  @wteouchicago what do you think?